modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -14,6 +14,7 @@ class UploadProcessor
     SUBMIT_DOC_PART_NAME = 'document'
     REQUIRED_KEYS = %w[veteranFirstName veteranLastName fileNumber zipCode].freeze
     FILE_NUMBER_REGEX = /^\d{8,9}$/
+    MAX_PART_SIZE = 100_000_000 # 100MB
```

Comment:
Hmm, I wonder if the limit is really 100 MB or 100 MiB. This is more conservative, so 👍

Reply:
I figured it was less likely to surprise us this way.